### DISCUSSION TITLE
Need some assistance with ./gradlew check

### DISCUSSION THREAD

**Discussion by akshay11agrawal** [micronaut-projects/micronaut-core#10642] [Q&A]
Hi Team,

I am new here in the community and interested in contributing to this project. I have taken the latest code for the 4.4.x version, however facing some issues while running gradlew check command:

`Execution failed for task ':buffer-netty:japiCmp'.
> A failure occurred while executing me.champeau.gradle.japicmp.JApiCmpWorkAction
   > Detected binary changes.
         - current: buffer-netty-4.4.0-SNAPSHOT.jar
         - baseline: micronaut-buffer-netty-4.3.12.jar.
`
Note: I have not made any changes to the codebase

Followed the readme and contribution guidelines/commands, any lead will be very helpful. Thanks

**Comment by sdelamo** [comment]
try with `./gradlew check -x japicmp`. japiCmp is a task we use to detect binary compatibility changes.
